devtools/project-core-extension-codestarts/src/main/resources/codestarts/quarkus/extension-codestarts/reactive-messaging-codestart/base/src/main/resources/application.tpl.qute.yml

```diff
@@ -0,0 +1,163 @@
+{#each input.selected-extensions-ga}
+  {#if it.equals("io.quarkus:quarkus-smallrye-reactive-messaging-kafka")}
+    {#if count == 1 }
+
+kafka:
```

Comment:
It is better to remove this for the dev services to create a Kafka broker

Reply:
fixed